src/test/java/reposense/git/GitDiffTest.java

```diff
@@ -31,4 +31,11 @@ public void diffCommit_latestCommitHash_emptyResult() {
     public void diffCommit_nonexistentCommitHash_throwsRunTimeException() {
         GitDiff.diffCommit(config.getRepoRoot(), NONEXISTENT_COMMIT_HASH);
     }
+
+    @Test
+    public void diffCommit_commitContainingSubmodule_ignoresSubmodule() {
```

Comment:
please keep the branch name the same as method name
E.g. IssueNumber-JavaClass-MethodName

Reply:
I changed the name of the branch the branch pushed up.
However, the branches are protected and I could not delete the old branch, as well as another branch i accidentally pushed up with a typo.
Can you give me permission to delete these branches?

Reply:
all the branches are protected by default, hence only the admins can delete them.
Can just let any of the admins know `@reposense/admins` and we will delete them for you

Reply:
Thank for the information :)